packages/rum/test/e2e/integration-test.js

```diff
@@ -73,6 +73,8 @@ async function runIntegrationTest(pageUrl) {
     result.response = {
       status: transactionResponse.status()
     }
+  } catch (err) {
+    console.log('[Intgration Tests Error]', err)
```

Comment:
Show re-throw the error as well!

Reply:
I dont think so, we will are back to the same unhanded rejection and would break the flow again in here https://github.com/elastic/apm-agent-rum-js/blob/87bbf600c84f8832a79e78457d43bafc0d1a6f7b/packages/rum/run-script.js#L90-L97.

We should catch and log the error and simply send a empty result which should break the tests.

Reply:
I have done a proper fix now according to how our integration test env requires and also fixed the timeout caused by not calling done